easybuild/easyconfigs/b/BLAST+/BLAST+-2.2.30-goolf-1.4.10.eb

```diff
@@ -0,0 +1,40 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2013 University of Luxembourg/Luxembourg Centre for Systems Biomedicine
+# Authors::   Fotis Georgatos <fotis.georgatos@uni.lu>, Kenneth Hoste (UGent)
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'BLAST+'
+version = "2.2.30"
+
+homepage = 'http://blast.ncbi.nlm.nih.gov/'
+description = """Basic Local Alignment Search Tool, or BLAST, is an algorithm
+ for comparing primary biological sequence information, such as the amino-acid
+ sequences of different proteins or the nucleotides of DNA sequences."""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+sources = ['ncbi-blast-%(version)s+-src.tar.gz']
+source_urls = ['ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/%(version)s/']
```

Comment:
not sure if this is a valid remark, but the from-pr fails for me because we can't download over ftp on our site, I see the http version of the url also works ;-)